test/functional/services/data_grid.ts

```diff
@@ -26,7 +26,7 @@ interface SelectOptions {
 export class DataGridService extends FtrService {
   private readonly find = this.ctx.getService('find');
   private readonly testSubjects = this.ctx.getService('testSubjects');
-  private readonly header = this.ctx.getPageObject('header');
+  private readonly docTable = this.ctx.getService('docTable');
```

Comment:
`docTable` service is for the legacy table view and it will be removed at some point. Looks like `addInclusiveFilter` and `removeInclusiveFilter` are not being used at all and can be removed from this file.

Reply:
indeed, fixed in 4b0e28e573e4e60ccb038c4046a01a703e65a11f